uniffi_core/src/metadata.rs

```diff
@@ -80,7 +80,7 @@ pub mod codes {
     pub const LIT_NULL: u8 = 4;
 }
 
-const BUF_SIZE: usize = 4096;
+const BUF_SIZE: usize = 16384;
```

Comment:
Let me know if you wanna use `8196` instead, and then I will need to decrease the amount of variants in the large error enum.

Reply:
Do you mind adding a semi-terse comment here linking to the 2 issues you linked in the new fixture?

Reply:
Fixed, check if wording is ok :)